src/messages.ts

```diff
@@ -23,6 +24,24 @@ const Messages = {
 
   },
 
+  async retry ( file: vscode.Uri ) {
+    const option = await vscode.window.showWarningMessage (
+      'Installation failed, would you force installation.',
+      { title: 'This time' },
+      { title: 'Always' }
+    );
+
+    if ( !option ) {
+      return;
+    } else if ( option.title === 'This time' ) {
+      vscode.commands.executeCommand( 'installVSIX.install', file, true );
+    } else if ( option.title === 'Always' ) {
+      await Utils.setInstallationForced ( true );
```

Comment:
I don't like this, the user should set the toggle manually if he wanted it, perhaps instead of changing the settings for them we could tell them what setting to set.

Reply:
I have change the options.